Lib/traceback.py

```diff
@@ -1520,8 +1520,9 @@ def _compute_suggestion_error(exc_value, tb, wrong_name):
         # has the wrong name as attribute
         if 'self' in frame.f_locals:
             self = frame.f_locals['self']
-            if hasattr(self, wrong_name):
-                return f"self.{wrong_name}"
+            if frame.f_code.co_name != '__getattr__':
```

Comment:
This also happens with `__getattribute__`.

Reply:
Addressed 05a985851955ea8c3508818d1e9f700d3248a7bf